src/ray/core_worker/lib/java/jni_utils.h

```diff
@@ -210,6 +210,14 @@ extern JavaVM *jvm;
     }                                                                        \
   }
 
+/// Throw a Java RuntimeException if the status is not OK.
+#define THROW_RUNTIME_EXCEPTION_IF_NOT_OK(env, status)                       \
```

Comment:
Why not reuse `THROW_EXCEPTION_AND_RETURN_IF_NOT_OK`?

Reply:
because it's invoker is a return `void` function,  it will compile failed  if invoke `THROW_EXCEPTION_AND_RETURN_IF_NOT_OK` function, I actually have a question here,  what's the meaning of `return` if throw a java `RuntimeException`

Reply:
You can use `THROW_EXCEPTION_AND_RETURN_IF_NOT_OK(env, status, (void)0)` for return-void functions.

Because a Java exception is not a C++ exception, calling `ThrowNew` does not throw a C++ exception. We have to call `return` manually to abort the execution of further lines of code of this function.

Reply:
updated.